bin/save-manifest-assets.sh

```diff
@@ -33,6 +33,8 @@ while read -r line; do
     fi
     kind=$(echo $line | awk '{ print $1 }')
 
+    echo "LINE $line"
```

Comment:
Debug?

Reply:
No its helpful to know where it fails. I'd rather leave it in there.